application/laravel/welcome.blade.php

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html lang="{{ app()->getLocale() }}">
+<head>
+<title>Hello World</title>
+</head>
+<body>
+<h1> Username: {{ env('APP_USERNAME') }}</h1>
+<h1> Password: {{ env('APP_PASSWORD') }}</h1>
```

Comment:
I think it would be nicer and more consistent with other examples if we displayed a "Hello {{username}}" / "Credentials not supplied" message here.

Reply:
Fixed, let me know if this is better.